freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/keytab/KerberosMgmtV1Endpoint.java

```diff
@@ -0,0 +1,49 @@
+package com.sequenceiq.freeipa.api.v1.keytab;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabModelNotes;
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabRequest;
+import com.sequenceiq.freeipa.api.v1.keytab.model.ServiceKeytabResponse;
+import com.sequenceiq.freeipa.api.v1.keytab.model.UserKeytabRequest;
+import com.sequenceiq.freeipa.api.v1.keytab.model.UserKeytabResponse;
+import com.sequenceiq.freeipa.api.v1.keytab.doc.KeytabOperationsDescription;
+import com.sequenceiq.service.api.doc.ContentType;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Path("/v1/kerberosmgmt")
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(value = "/v1/kerberosmgmt", protocols = "http,https")
+public interface KerberosMgmtV1Endpoint {
+
+    @GET
```

Comment:
Should this API be split into a POST and a GET?
I am thinking that the POST will alter state (ie create the host, service principal, and keytab) while the GET will only get an existing keytab and the GET will then be stateless.

Reply:
I agree. There should API for each. POST should create necessary entities and GET should fetch it.